rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -196,6 +196,20 @@ void ObjectRenderer::renderInstance(InstanceObject *instance,
 		return;
 	}
 
+
```

Comment:
nit: A comment stating that this handles times supplied by TOBJ would be nice imo